settings/installer/ospd/overcloud/storage/ceph-external.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
As mentioned in the spec, I think it will be better to change the structure so instead of "settings/ospd/overcloud" we will have settings/ospd/{storage, network, etc..}